docs/components/toaster.md

```diff
@@ -64,192 +57,207 @@ If a global property conflicts with a component’s own property, the component'
 
 ## Arguments
 
-### message
+KToaster is the underlying component rendered by the `ToastManager` instance, so all component properties are passed down via `ToastManager.open()` methods' arguments. The accepted argument type is `string` or object that is instance of `Toast`.
 
-The default argument passed to the toaster is the message.
+```ts
+interface Toast {
+  key?: any
+  title: string
+  message?: string
+  appearance?: ToasterAppearance
+  timeoutMilliseconds?: number
+}
+```
 
-<KButton @click="$toaster.open('Default message here')">Open Toaster</KButton>
+### title
+
+Notification title. When passing an object to `$toaster.open()` method, the `title` property is required. When passing a string, it will be rendered as title.
+
+<div class="horizontal-container">
+  <KButton @click="$toaster.open({ title: 'Long Title Gets Truncated With Ellipsis' })">Open Toaster</KButton>
+  <KButton @click="$toaster.open('String Will Become A Title')" appearance="secondary">Open Toaster</KButton>
+</div>
 
 ```html
-<KButton @click="$toaster.open('Default message here')">Open Toaster</KButton>
+<KButton @click="$toaster.open({ title: 'Long Title Gets Truncated With Ellipsis' })">Open Toaster</KButton>
+<KButton @click="$toaster.open('String Will Become A Title')" appearance="secondary">Open Toaster</KButton>
 ```
 
-### appearance
+### message
 
-The Toaster uses the same appearance values as [KAlert](/components/alert) and are applied the same way.
+Because a long title gets truncated, it is only good for short notifications. When a longer notification needs to be shown to the user, `message` property becomes helpful.
```

Comment:
```suggestion
The message prop allows for displaying longer strings of text to the user. This prop is good for more detailed messages, or displaying IDs, etc.

Because a long title gets truncated, it is only good for short notifications. When a longer notification needs to be shown to the user, the `message` property is ideal.
```